### PR TITLE
test: add tests for issue 2295

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

**Comment by JoanFM** [comment]
Test with proposed quick fix for #2295

**Comment by cristianmtr** [comment]
Closes #2295  ?

**Comment by JoanFM** [comment]
> Closes #2295 ?

May have a quick fix for 2295, but the underlying problem is in #2299

File: tests/integration/issues/github_2295/test_issue.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from jina.flow import Flow
+from jina import Document, DocumentSet
+
+from tests import validate_callback
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+class SearchDocIndexer(BinaryPbIndexer):
+    def query(self, jina_id: str = None, mongo_ids: List[str] = None):
```
Comment (cristianmtr) [inline tests/integration/issues/github_2295/test_issue.py:17]: Why only 1 `jina_id` but multiple `mongo_ids`?

File: tests/integration/issues/github_2295/test_issue.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        return super().query(jina_id)  # serialized document
+
+    def post_init(self):
+        super().post_init()
+        self.name = 'doc_idx_file'
```
Comment (cristianmtr) [inline tests/integration/issues/github_2295/test_issue.py:22]: maybe make a comment for this to explain why it was necessary and link to the issue that describes the underlying problem

File: tests/integration/issues/github_2295/test_issue.py (hunk 1 above)
Comment (JoanFM) [reply tests/integration/issues/github_2295/test_issue.py:17]: this is provided by the client, is just an example

File: tests/integration/issues/github_2295/test_issue.py (hunk 2 above)
Comment (JoanFM) [reply tests/integration/issues/github_2295/test_issue.py:24]: good point, done

**Review by cristianmtr — APPROVED** [review]

**Pull request merged**